Dev/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js

```diff
@@ -98,7 +105,9 @@ export class ClassMainScreen extends QcParentScreen {
               <TopBanner
                 LeftIconName="navicon"
                 LeftOnPress={() => this.setState({ isOpen: true })}
+                isEditingTitle={this.state.isEditing}

```

Comment:
I believe that this if clause is when there is not class currently being viewed. In this case, isEditingTitle should always be false. Right? Or am I misunderstanding